docs/guide.rst

```diff
@@ -2,4 +2,76 @@
 User Guide
 **********
 
-This user guide provides a walkthrough of the major features in the ``hvpy`` package.
+Welcome to ``hvpy``!
```

Comment:
```suggestion
```
I feel like this should be on the main index page?

Reply:
Yea, that's okay. Removed.